nl.emphisia.icon.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id" : "nl.emphisia.icon",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "46",
+    "sdk" : "org.gnome.Sdk",
+    "runtime-version" : "46",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "folder_icon",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a",
+    ],
+    "modules" : [
+	{
+	  "name": "blueprint-compiler",
+	  "buildsystem": "meson",
+	  "cleanup": ["*"],
+	  "sources": [
+	    {
+	      "type": "git",
+	      "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+	      "tag": "v0.12.0"
+	    }
+	  ]
+	},
+        {
+            "name" : "folder_icon",
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type": "archive",
+                    "url" : "https://github.com/youpie/Iconic/releases/download/v1.0.0/Iconic-Release.tar.xz",
```

Comment:
Did you add a homepage to metainfo ? https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#url

Reply:
i did not because i dont have a homepage for the app, what should I use as homepage?

Reply:
I have just used `emphisia.nl`

Reply:
Is the website up? Can't seem to reach it.

Reply:
I can reach it no problem? does it work if you explicitly type `https://`

Reply:
It was a DNS issue on my end.